src/BUILD.gn

```diff
@@ -30,7 +30,7 @@ declare_args() {
 
 # chip_root relative to root_build_dir for macro-prefix-map.
 # "/." avoids trailing "/" in result when chip_root is "//"
-build_relative_chip_root = rebase_path("${chip_root}/.", root_build_dir)
+# build_relative_chip_root = rebase_path("${chip_root}/.", root_build_dir)
```

Comment:
do not comment out things, remove them instead.

I believe these are intentional for non-android at least (and android should benefit from them too presumably).

If android really fails to build, then having a current_os check for android may work ... however our github CI passes, so I am unsure what is going on here.